runner.go

```diff
@@ -386,6 +395,38 @@ func (r *LocalRunner) ContainerRunner(step Step, network Network) func() error {
 	}
 }
 
+// ContainerLogReader returns a function to ...
+func (r *LocalRunner) ContainerLogReader(step Step, follow bool) func() error {
+	return func() error {
+		if Verbose {
+			log.Printf("Opening logs for container '%s'", step.ContainerName())
+		}
+		r.prefix = step.ColoredContainerName()
+		args := []string{"logs"}
+		if follow {
+			args = append(args, "-f")
+		}
+
+		// Get id(s) of container with name of step
+		out, err := r.Output([]string{"ps", "-a", "-q", "--filter", fmt.Sprintf("name=%s$", step.ContainerName())})
```

Comment:
This ought be duplicate at multiple places, maybe turn it into a helper function

Reply:
Will do, good idea.